### ISSUE TITLE
Issue with building modified Clang with llvm 

### ISSUE THREAD

**Issue by yashie2011** [uwsampa/accept#48]
I have been trying to install accept compiler on a ubuntu 16.04 machine. I run the command "make setup" after installing Cmake, Ninja and virtual-env. I get the following error which I am not sure how to debug. 

cd llvm/tools ; ln -s ../../clang .
ln: failed to create symbolic link './clang': Operation not supported
Makefile:103: recipe for target 'llvm/tools/clang' failed

Thank you. 
Yaswanth

**Comment by sampsyo** [comment]
That's interesting! Is there anything strange about the filesystem you're running this on? Some quick googling suggests that this might happen on a hypervisor's shared volumes, for example: https://communities.vmware.com/thread/312591

**Comment by yashie2011** [comment]
Yes. The compiler source code was located in a shared/network volume, and the root file system is in local drive. Installation was successful on a standalone drive.  

But I might be running out of disk space. It doesn't work If I want to save my installation on a network drive. Not a high priority issue but worth trying for a fix. 

Thank you.

**Comment by sampsyo** [comment]
OK! I’m not sure what to suggest, though, if your FS doesn’t support symlinks (as most do). Maybe there’s a way to configure your hypervisor to allow symlinks?

**Comment by yashie2011** [comment]
Maybe. I will look into that.

Thank you

**Comment by umair626** [comment]
Hi, 
I want to install ACCEPT and I got into the following error after running make setup

umair@umair-pc:~/accept$ make setup
/bin/sh: 1: ninja: not found
Please install Ninja to build LLVM and ACCEPT.
http://martine.github.io/ninja/
Makefile:118: recipe for target 'check_ninja' failed
make: *** [check_ninja] Error 2

Any help will be appreciated.thanks
